Modules/timemodule.c

```diff
@@ -1840,11 +1822,34 @@ PyInit_time(void)
         utc_string = tm.tm_zone;
 #endif
 
-    return m;
+    return 0;
 
 error:
-    Py_DECREF(m);
-    return NULL;
+    Py_DECREF(module);
```

Comment:
DECREF here is wrong.

cc @corona10

Reply:
I started not wanting to refactor code but looks like a good chance for more cleaning. Thanks, already changed on latest push.